MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -62,18 +62,17 @@
                             <use href="img/icons.svg#icon-heart"></use>
                         </svg>
                     </div>
-                    <div class="likes__panel">
-                        <ul class="likes__list">
-
-                            @if (likedRecipes != null)
-                            {
+                    @if (likedRecipes != null)
+                    {
+                        <div class="likes__panel">
+                            <ul class="likes__list">
 
-                                foreach (var recipe in likedRecipes)
+                                @foreach (var recipe in likedRecipes)
                                 {
                                     <li>
                                         <a class="likes__link" href="#@recipe["id"]">
                                             <figure class="likes__fig">
-                                                <img src="@recipe["image"]" alt="Test">
+                                                <img src="@recipe["image"]" alt="Recipe">
```

Comment:
Alt text is good, but Recipe is only marginally more effective than test.  Expand on this concept.  What is the recipe about?  What is in it?